crates/postgres-subsystem/postgres-model/src/types.rs

```diff
@@ -177,6 +177,12 @@ pub struct PostgresField<CT> {
     pub dynamic_default_value: Option<ContextSelection>,
     pub readonly: bool,
     pub access: Access,
+    pub type_props: Option<TypeProps>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum TypeProps {
```

Comment:
Let's rename this to something like: `TypeValidation` or `TypeValidationInfo` or `ValidationInfo`